doc/contributing/development-environment.md

```diff
@@ -28,46 +39,35 @@ This will give you the `master` branch of TerriaJS.  While we strive to keep `ma
 ```
 grep terriajs package.json
 # will print something like: "terriajs": "^4.5.0"
-cd node_modules/terriajs
+cd packages/terriajs
 git checkout 4.5.0
-cd ../..
-```
-
-Now, if you run `npm install`, `npm` will fail with an error like this:
-
-```
-npm ERR! git C:\github\TerriaMap\node_modules\terriajs: Appears to be a git repo or submodule.
-npm ERR! git     C:\github\TerriaMap\node_modules\terriajs
-npm ERR! git Refusing to remove it. Update manually,
-npm ERR! git or move it out of the way first.
+cd ..
 ```
 
-This is _good_!  It means `npm` recognizes that your `terriajs` directory is now a git repo, and it is refusing to touch it out of fear of making you lose your work.  This is where `npmgitdev` comes in:
+Then, in the TerriaMap directory, run:
 
 ```
-npmgitdev install
+yarn install
 ```
 
-`npmgitdev` is a wrapper around `npm` that, for each git repo in `node_modules`:
+Yarn will:
+* Install all dependencies for both TerriaMap and any packages in your `packages` directory.
+* Install the `devDependencies` for packages in the `packages` directory so you can actually develop on them.
+* Create sym-links so that everything works.
+* De-duplicate semver-compatible packages and bubble them up to the root `node_modules` directory.
 
-* Makes sure your working directory is clean (i.e. you have no uncommitted changes).
-* Moves the `.git` directory out of the way so that `npm` can't see it or clobber it.
-* Copies all the `devDependencies` in `package.json` into `dependencies`.  This way npm will install all your dev-time stuff too.
-* Runs `npm` normally.
-* Moves the `.git` directory back.
+Now, we can edit TerriaJS in `packages/terriajs` with the benefit of a full-featured git repo.
 
 !!! note
 
-	If you hit CTRL-C while `npmgitdev` is running, it will be unable to clean up after itself automatically.  Look for a `npmgitdev-XXXXX` directory.  The `mappings.json` file in that directory contains a record of the changes that `npmgitdev` made so that you can undo them manually.  Usually this just means moving the `.git` directory back.
-
-Now, we can edit TerriaJS in `node_modules/terriajs` with the benefit of a full-feature git repo.
+	Running `yarn install` with workspaces configured will change `yarn.lock`.  Please do not commit the changes.
```

Comment:
Maybe should mention in *Committing modifications* section below on whether these changes should be committed (I assume not?)?